Cargo.toml

```diff
@@ -1,22 +1,27 @@
 [package]
-name = "rar"
-version = "0.3.0"
-authors = ["Robert Schütte <schuetterobert@gmail.com>"]
+name = "rar-rs"
+version = "0.4.0"
+authors = ["Robert Schütte <schuetterobert@gmail.com>", "Michael Stowe <michael.stowe@member.mensa.org>"]
 license = "MIT"
 readme = "README.md"
-repository = "https://github.com/Roba1993/RAR"
-homepage = "https://github.com/Roba1993/RAR"
+repository = "https://github.com/mwstowe/RAR"
+homepage = "https://github.com/mwstowe/RAR"
 description = """
-edition= "2018"
 Rust native RAR extractor based upon nom
 """
+edition = "2021"
 exclude = [
     "assets/*",
 ]
 
 [dependencies]
-nom = "4.0.0"
-failure = "0.1.1"
-lazy_static = "1.1.0"
-chrono = "0.4.5"
-rust-crypto = "0.2.36"
\ No newline at end of file
+nom = "7.1"
+anyhow = "1.0"
```

Comment:
Because this is a library, anyhow should be replaced with https://docs.rs/thiserror/latest/thiserror/